build.gradle

```diff
@@ -1,68 +1,98 @@
-buildscript {
-	dependencies {
-		classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0'
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+	alias(libs.plugins.kotlin.jvm)
+	alias(libs.plugins.spotless)
+}
+
+kotlin {
+	jvmToolchain(21)
+}
+
+tasks.withType(KotlinCompile).configureEach {
+	compilerOptions {
+		jvmTarget = JvmTarget.JVM_11
 	}
 }
 
-apply plugin: 'org.jetbrains.kotlin.jvm'
+tasks.withType(JavaCompile).configureEach {
+	options.release.set(11)
+}
```

Comment:
AFAIU these two should be kept in sync, but I don't see any easy way to use single variable. 
Moreover, I haven't found target version defined explicitly anywhere, so I used Java 11 which was defined in the Github Workflow file